src/firefly/js/templates/lightcurve/LcResult.jsx

```diff
@@ -124,52 +142,78 @@ const StandardView = ({visToolbar, title, searchDesc, imagePlot, xyPlot, tables,
     const cutoutSize = get(settingBox, 'props.generalEntries.cutoutSize', '0.3');
     const converterId = get(settingBox, ['props', 'missionEntries', LC.META_MISSION]);
     const mission = getMissionName(converterId) || 'Mission';
+    const showImages = isEmpty(imagePlot);
+
+    var tsView = (err) => {
+
+        if (!err) {
+            return (
+                <SplitPane split='horizontal' maxSize={-20} minSize={20} defaultSize={'60%'}>
+                    <SplitPane split='vertical' maxSize={-20} minSize={20} defaultSize={buttonW}>
+                        <SplitContent>
+                            <div style={{display: 'flex', flexDirection: 'column', height: '100%'}}>
+                                <div className='settingBox'>{settingBox}</div>
+                                <div style={{flexGrow: 1, position: 'relative'}}>
+                                    <div className='abs_full'>{tables}</div>
+                                </div>
+                            </div>
+                        </SplitContent>
+                        <SplitContent>{xyPlot}</SplitContent>
+                    </SplitPane>
+                    <SplitContent>{imagePlot}</SplitContent>
+                </SplitPane>
+            );
+        } else {
+            return (
+                <SplitPane split='vertical' maxSize={-20} minSize={20} defaultSize={buttonW}>
+                    <SplitContent>
+                        <div style={{display: 'flex', flexDirection: 'column', height: '100%'}}>
+                            <div className='settingBox'>{settingBox}</div>
+                            <div style={{flexGrow: 1, position: 'relative'}}>
+                                <div className='abs_full'>{tables}</div>
+                            </div>
+                        </div>
+                    </SplitContent>
+                    <SplitContent>{xyPlot}</SplitContent>
+                </SplitPane>
```

Comment:
The table/chart <SplitPane...../> is the same in both cases.. you should reuse it, instead of duplicating it,